tests/framework/user_guide/AdaptiveSamplingStrategies/adaptiveSamplingLSsearch.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<Simulation verbosity="debug">
+<Simulation verbosity="silent">
```

Comment:
This might be somewhat painful to fix if the test fails; was there an issue with the verbosity?

Reply:
For this test, the debug output will be very large, and civet can not even show the output of this test. When I download the test result, the output also trimmed.